lib/internal/util/inspect.js

```diff
@@ -1573,7 +1573,7 @@ function handleMaxCallStackSize(ctx, err, constructorName, indentationLvl) {
 function addNumericSeparator(integerString) {
   let result = '';
   let i = integerString.length;
-  const start = StringPrototypeStartsWith(integerString, '-') ? 1 : 0;
+  const start = integerString[0] === '-' ? 1 : 0;
```

Comment:
```suggestion
  const start = i && integerString[0] === '-' ? 1 : 0;
```
Alternatively, we could simply have an assert, as there's no usecase I can think of for passing an empty string to that util function
```suggestion
  assert(i !== 0);
  const start = integerString[0] === '-' ? 1 : 0;
```

Reply:
Yeah I checked its usage and it's only passed numbers that get coerced to strings, so impossible to have empty strings

But don't know if it's better to always check in these cases should someone accidentally pass an empty string in the future

Reply:
IMO that's what the `assert` is for, to ensure the usage doesn't change without us noticing